### DISCUSSION TITLE
Week 2 Report

### DISCUSSION THREAD

**Discussion by AndresMasis** [SE-TINF22B6/CookPit#16] [General]
**Team:** CookPit

**Members:** Dominik Nedele (Product Owner)
Karim Gharbi (SCRUM Master)
Simon Hajek
Robin
Andres Masis

**Roles:** All the memebers can collaborate in tasks from all roles (desgin, backend, frontend, databse). The initial design was made by the whole team, Andres is more focused on the database, the rest of the team is more focused on the front end. The backend and documentation is handled by all the team.

**Vision:** We desire to create a recipe portal where our users have an unique cooking expereinces. We want to archieve a website where the user can easily find their favorite recipes and get amused by new flavors.

**Platform:** For the frontend we are going to use React and Bootstrap, for the backend we are going to use Node.js. For the database we will use PostgreSQL. We are also going to rely on the OpenAI API.

**Scrum:** The communication methods will be face to face during the class and online by Discord. Discord allows to create channels, share files, a powerful screen sharing and save chat, besides the normal video calls (that have no time limit in this platform). There will be a 30 min Scrum meeting every 2 weeks at Saturday afternoon. And every day there will be concise but constant communication in the Discord chat to see how is the team doing. The sprint was set for 2 weeks.

**Sprint 1 Backlog:** For this backlog, we  discussed how to manage the branches of the Github repository. We also discussed how the UI should look. We also discussed the design of the software architecture. In addition, we discussed the tech stack. We also arranged everything for the SCRUM working environment (sprints, team organization ,etc).

**Comment by Stev3eM** [comment]
Your Product Owner and Scrum Master role is missing as the screenshot of your Kanban. Furthermore a good first post.

Your CookHub team :-*

**Comment by giomskii** [comment]
As Stev3eM already mentioned, the precise roles and the kanban board are missing. Nonetheless, you described your vision, use of platforms/technologies and plans for the first sprint well 👍

**Comment by AndresMasis** [comment]
Thanks for your comments. The roles are updated and the teacher said the Kanban screenshot is not necessary because everyone can access that.

**Comment by ghost** [comment]
Team CookPit appears to have a well-rounded and collaborative approach to their project, and their vision of creating a unique recipe portal is both promising and exciting. I think the different skillsets of the team members can contribute to a great outcome of the recepie website. Im excited to see the progress and results ;)
